src/parser/jsoncons/json_parser.hpp.orig

```diff
@@ -0,0 +1,2157 @@
+// Copyright 2015 Daniel Parker
```

Comment:
This file has numerous conflicts in it.  If we aren't using it, then let's remove it.